web/elm/src/Dashboard/Grid/Constants.elm

```diff
@@ -21,17 +21,27 @@ cardBodyHeight =
 
 padding : number
 padding =
-    25
+    32
+
+
+carHeaderVerticalSpace : number
+carHeaderVerticalSpace =
+    8
 
 
 cardHeaderHeight : number -> number
 cardHeaderHeight numRows =
-    20 + 30 * numRows
+    32 + 21 * numRows + (numRows - 1) * carHeaderVerticalSpace
+
+
+groupHeaderLineHeight : number
+groupHeaderLineHeight =
+    25
```

Comment:
Can this be used in the CSS for the header?

Reply:
do you mean add a line-height 25px here ?https://github.com/concourse/concourse/blob/398ef03bbf45044912a3e8fda85200ba3ebb9480/web/assets/css/dashboard.less#L57

Without it the header line height is still 25px by the enforcement from (height 57px - margin-bottom 32px). Do you just want to explicity set it in CSS as well?